src/entity-manager/EntityManager.ts

```diff
@@ -161,17 +161,17 @@ export class EntityManager {
     /**
      * Creates a new query builder that can be used to build a sql query.
      */
-    createQueryBuilder<Entity>(entityClass: ObjectType<Entity>, alias: string, queryRunner?: QueryRunner): SelectQueryBuilder<Entity>;
```

Comment:
I don't think we should have `= any` here. What is the motivation behind this change?

Reply:
> I don't think we should have `= any` here. What is the motivation behind this change?

Yes, I think it should be unknown, but many tests fall

Reply:
I don't think we should put a default type at all, what can be a motivation? It looks like a monkey patching instead of root problem resolving solution.

Reply:
In the latest typescript version, for example in this block `createQueryBuilder<Entity>`, `Entity` by default will be `unknown` (in the current version used it is interpreted as `any`)
You can just upgrade the typescript version and see the difference